### ISSUE TITLE
Encoding bug in Lint/PercentStringArray

### ISSUE THREAD

**Issue by toy** [bbatsov/rubocop#5809]
Given file `test.rb`:
```ruby
# encoding: BINARY

%W[\xC0]
```

--------

## Expected behavior

`rubocop` would run without complaint.

## Actual behavior

`rubocop` failed with `An error occurred while Lint/PercentStringArray cop was inspecting …/test.rb`

## Steps to reproduce the problem

Create `test.rb` file, run `rubocop`.

## RuboCop version

```
0.55.0 (using Parser 2.5.0.5, running on ruby 2.5.1 x86_64-darwin16)
```
